src/vm-repair/azext_vm_repair/custom.py

```diff
@@ -316,11 +319,11 @@ def restore(cmd, vm_name, resource_group_name, disk_name=None, repair_vm_id=None
         # Fetch source and repair VM data
         source_vm = get_vm(cmd, resource_group_name, vm_name)
         is_managed = _uses_managed_disk(source_vm)
-        if repair_vm_id:
```

Comment:
why remove this if check?

Reply:
the statements were redundant due to how the variables where initialized and used, as well as the validator function not allowing a non-triggering condition in the first place.

Reply:
I will investigate this further, but I have called this command without the repair_vm_id before, so please revert this edit for your feature. I will edit this later.